src/services/account-states/config.ts

```diff
@@ -0,0 +1,160 @@
+    ],
+  },
+  FRAUD_BLOCK_ACCOUNT: {
+    code: 3,
+    state: {
+      blocked: true,
+      suspended: false,
+      resetPassword: false,
+      reproveIdentity: false,
+    },
+    allowedTransitions: [EventsEnum.FRAUD_UNBLOCK_ACCOUNT],
+    interventionName: AISInterventionTypes.AIS_ACCOUNT_BLOCKED,
+  },
+  FRAUD_UNBLOCK_ACCOUNT: {
+    code: 7,
+    state: {
+      blocked: false,
+      suspended: false,
+      resetPassword: false,
+      reproveIdentity: false,
+    },
+    allowedTransitions: [
+      EventsEnum.FRAUD_SUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_BLOCK_ACCOUNT,
+      EventsEnum.FRAUD_FORCED_USER_IDENTITY_REVERIFICATION,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_REVERIFICATION,
+    ],
+    interventionName: AISInterventionTypes.AIS_ACCOUNT_UNBLOCKED,
+  },
+  FRAUD_FORCED_USER_PASSWORD_RESET: {
+    code: 4,
+    state: {
+      blocked: false,
+      suspended: true,
+      resetPassword: true,
+      reproveIdentity: false,
+    },
+    allowedTransitions: [
+      EventsEnum.FRAUD_BLOCK_ACCOUNT,
+      EventsEnum.FRAUD_UNSUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_SUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_FORCED_USER_IDENTITY_REVERIFICATION,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_REVERIFICATION,
+    ],
+    interventionName: AISInterventionTypes.AIS_FORCED_USER_PASSWORD_RESET,
+  },
+  FRAUD_FORCED_USER_IDENTITY_REVERIFICATION: {
+    code: 5,
+    state: {
+      blocked: false,
+      suspended: true,
+      resetPassword: false,
+      reproveIdentity: true,
+    },
+    allowedTransitions: [
+      EventsEnum.FRAUD_BLOCK_ACCOUNT,
+      EventsEnum.FRAUD_UNSUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_SUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_REVERIFICATION,
+    ],
+    interventionName: AISInterventionTypes.AIS_FORCED_USER_IDENTITY_VERIFY,
+  },
+  FRAUD_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_REVERIFICATION: {
+    code: 6,
+    state: {
+      blocked: false,
+      suspended: true,
+      resetPassword: true,
+      reproveIdentity: true,
+    },
+    allowedTransitions: [
+      EventsEnum.FRAUD_BLOCK_ACCOUNT,
+      EventsEnum.FRAUD_UNSUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_SUSPEND_ACCOUNT,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET,
+      EventsEnum.FRAUD_FORCED_USER_IDENTITY_REVERIFICATION,
+    ],
+    interventionName: AISInterventionTypes.AIS_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_VERIFY,
+  },
+};
+
+export const userLedActionsConfig: UserLedActionTransitionConfigurations = {
+  IPV_IDENTITY_ISSUED: {
+    code: 8,
+    state: {
+      reproveIdentity: false,
+    },
+    allowedFromStates: [
+      EventsEnum.FRAUD_FORCED_USER_IDENTITY_REVERIFICATION,
+      EventsEnum.FRAUD_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_REVERIFICATION,
+    ],
+  },
+  AUTH_PASSWORD_RESET_SUCCESSFUL: {
+    code: 9,
```

Comment:
lets make this codes 99 - to leave space for extra codes from the rsik engine to be added without causing a clash